pages/docs/submitting.mdx

```diff
@@ -0,0 +1,172 @@
+work. Each challenge you solve is simulating building a feature at work.
+
+If this is all new to you, these steps may seem very intimidating. When you
+start a new job, you will be equally intimidated because of all the specific
+tools each company uses. It is important to ask questions when you feel
+overwhelmed, there will be people on the chatroom help answer your questions.
+
+## Local Setup
+
+When you join a software engineering company, the first step is setting up your
+machine, downloading the codebase, and setting up the codebase.
+([video instructions](https://www.youtube.com/watch?v=JnRgR7z0rFU&t=254s))
+
+1. After installing `node`, install c0d3 CLI (used to submit challenges):
+   `npm i -g c0d3`
+
+   - If you run into a `Missing write access ... EACCES` error, you need to run
+     npm with sudo:
+     - `sudo npm i -g c0d3`
+
+1. Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+   and familiarize yourself with [common git commands](/docs/git)
+1. Use git to [clone the curriculum](https://github.com/garageScript/curriculum)
+   into your machine
+   - `git clone [https://github.com/garageScript/curriculum](https://github.com/garageScript/curriculum)`
+1. Go into the curriculum root folder by typing `cd curriculum`
+1. Download the necessary libraries by running `yarn install`
+
+## Browser Setup
+
+1. Go to
+   [GitPod](https://gitpod.io/#https://github.com/garageScript/curriculum).
+   You'll be asked to create an account or you can also login using a Github
+   account.
+2. After your workspace is created, you'll have most of the tools you need at
+   your disposal (`git`, `node`, `npm` and `yarn`). You'll need to install only
+   the c0d3 CLI tool used to submit the challenges, by typing `npm i -g c0d3` on
+   the terminal at the bottom of the screen. **Note:** every time your workspace
+   is restarted you'll need to run this command again.
+3. After installing the CLI tool, type `c0d3 login` and enter your C0D3 account
+   credentials to be able to submit the challenges.
+4. Now the only thing remaining is to download the necessary libraries by
+   running `yarn install`.
+
+## Solving Challenges
+
+`git status` is your best friend. It tells you what branch you are currently on
+and which files are changed and/or staged for commit.
+
+Solving each challenge is like building a feature at work. You must always
+checkout `checkout master` before creating a new branch to work on a new
+feature.
+
+Branch names: Usually branch names are for your own eyes only, so you do not
+have create branches with meaningful names.
+
+_Note: Except for steps 3 (reading the challenge) and 4 (open and editing the
+file), all the commands should be run in the terminal_
+
+1. Make sure you are on the master branch: `git checkout master`
+2. Create a new branch from master (pro tip: `p1` for problem 1)
+   - `git checkout -b p1`
+3. Go to [c0d3.com](http://c0d3.com), click on Curriculum, select lesson 0, and
```

Comment:
link to relative path or to `https://www.c0d3.com` to skip the redirection server.

```suggestion
3. Go to [c0d3.com](/), click on Curriculum, select lesson 0, and
```